lib/content-services/src/lib/search/components/search-filter-chips/search-widget-chip/search-widget-chip.component.ts

```diff
@@ -19,11 +19,18 @@ import { Component, ElementRef, Input, ViewChild, ViewEncapsulation } from '@ang
 import { SearchCategory } from '../../../models/search-category.interface';
 import { ConfigurableFocusTrap, ConfigurableFocusTrapFactory } from '@angular/cdk/a11y';
 import { MatMenuTrigger } from '@angular/material/menu';
-import { SearchWidgetContainerComponent } from '../../search-widget-container/search-widget-container.component';
+import { SearchWidgetContainerComponent } from '../../search-widget-container/search-widget-container.component';;
```

Comment:
Doubled `;`

Reply:
The changes in this file and its corresponding HTML file were no longer required after a recent change in the component structure. Reverted these files to their original state